src/DevTools/MockBoot.tsx

```diff
@@ -18,7 +27,7 @@ export const MockBoot: React.SFC<{
       headTags={headTags}
       context={mockContext as ClientContext}
       user={user}
-      relayEnvironment={null as any}
+      relayEnvironment={relayEnvironment ?? (null as any)}
```

Comment:
@damassi and this is why I also added this here, because without this I wasn't able to have access to the mutation that happens when we press the commercial buttons